libs/gl-client-py/glclient/__init__.py

```diff
@@ -88,13 +89,13 @@ def export_node(self) -> schedpb.ExportNodeResponse:
         res = schedpb.ExportNodeResponse
         return res.FromString(bytes(self.inner.export_node()))
 
-    def node(self) -> "Node":
+    def node(self, creds: Credentials) -> "Node":
```

Comment:
We probably want to have `Scheduler` have a copy of the `Credentials`, since we will eventually enforce that only an authenticated client can schedule. That would mean that the `schedule()` call also needs the `Credentials`, and we could just inherit from `Scheduler` to `Node` when creating the `Node` here.

Reply:
Let's talk about this during our meeting.